app/views/trainees/start_dates/edit.html.erb

```diff
@@ -1,14 +1,17 @@
-<%= render PageTitle::View.new(title: "trainees.trainee_ids.edit") %>
+<%= render PageTitle::View.new(title: "trainees.trainee_start_date.edit") %>
 
 <%= content_for(:breadcrumbs) do %>
   <%= render DynamicBackLink::View.new(@trainee, text: t(:back)) %>
 <% end %>
 
 <%= form_with model: @training_details, url: trainee_start_date_path, local: true do |f| %>
-  <%= f.govuk_date_field :commencement_date, legend: {
-    text: t("views.forms.training_details.commencement_date.label"), size: 'l', tag: "h1"
-  }, hint: { text: t("views.forms.training_details.commencement_date.hint", year: Time.zone.now.year)} %>
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
```

Comment:
The markup isn't quite right here.

The formbuilder method `govuk_date_field` creates the following under the hood:

```
<div class="govuk-form-group">
  <fieldset class="govuk-fieldset">
     ... and here's the date input  ...
  </fieldset>
</div>
```

The legend that's describing the date input, should be inside the same `<fieldset>` - here your legend is outside.

Reply:
Sticking with the formbuilder, it essentially creates the same markup under the hood when using legend option 
<img width="486" alt="Screenshot 2021-03-03 at 11 38 30" src="https://user-images.githubusercontent.com/58793682/109800494-257c6980-7c15-11eb-8130-c6c3ee79a0eb.png">
But without the CSS styling there is still a smaller margin, as there is an empty hint div on the prototype. 
<img width="420" alt="Screenshot 2021-03-03 at 11 43 28" src="https://user-images.githubusercontent.com/58793682/109800968-c539f780-7c15-11eb-8213-65088c698a2d.png">
The problem is, thinking about it, that it will create a 10px bottom margin on all of those across the service

Reply:
@JonF1 the prototype markup is mostly not a good _exact_ reference. It's a rough visual guide - but the markup is often horrible. It's probably got an empty hint because we've just commented it out lazily. Definitely default to the default formbuilder stuff.

Reply:
Ok great, thanks guys